### ISSUE TITLE
Firefox headless browser

### ISSUE THREAD

**Issue by NihadMemmedli** [golemhq/golem#121]
Is there any way for running on firefox headless mode?

**Comment by rothnic** [comment]
Hopefully this is of some help. There wasn't a lot of guidance on how to do this with a json config, typically you see lots of code examples. What I ended up doing was using python to build the config I know works, then used the `to_capabilities()` method on the options object to get the JSON representation.

So, this [Golem Selenium Grid Docker-Compose Example](https://gist.github.com/rothnic/79e036ce4564a3ecb2708e248d163ceb) includes my config for remote devices with a headless chrome laptop-sized device, a headless chrome mobile-sized device, then a headless firefox device.

One thing to note is that I'm using docker selenium grid nodes, and I had to share the host machines memory with the nodes, otherwise the firefox node would randomly timeout and error out. That is configured in the docker-compose file.
